abletonosc/track.py

```diff
@@ -39,13 +39,15 @@ def track_callback(params: Tuple[Any]):
                                         create_track_callback(self._set, prop))
 
         def track_get_volume(track, params: Tuple[Any] = ()):
-            return track.mixer_device.volume.value
+            params = (track.mixer_device.volume.value,)
+            return params
```

Comment:
Silly mistake on my part - clearly this callback has never worked! 
For consistency, could you switch this to the same syntax used in (e.g.) song.py:

```
        def song_get_num_tracks(song, params: Tuple[Any] = ()):
            return len(song.tracks),
```

Saves a line of code and the creation of a temporary `params` tuple.